test/files/run/t6329_vanilla.check

```diff
@@ -1,4 +1,4 @@
-scala.collection.immutable.List[<?>]
+scala.collection.immutable.List[Any]
```

Comment:
This one is specifically called out in #1900 as "That's correctness", so it's concerning this is changing back...

Reply:
I don't see how that's correctness. This is the `toString` of `ClassManifest` which is deprecated but I don't think in general it works for type parameters due to erasure?